proposal-templates/0000-swift-template.md

```diff
@@ -3,15 +3,39 @@
+
+`Status` should reflect the current implementation status while the proposal is still a PR.  The proposal cannot be reviewed until an implementation is available, but early readers should see the correct status.
+
+`Vision` should link to the [vision document](https://forums.swift.org/t/the-role-of-vision-documents-in-swift-evolution/62101) for this proposal, if it is part of a vision.  Most proposals are not part of a vision.  If a vision has been written but not yet accepted, link to the discussion thread for the vision.
+
+`Roadmap` should link to the discussion thread for the roadmap for this proposal, if applicable.  When a complex feature is broken down into several closely-related proposals to make evolution review easier and more focused, it's helpful to make a forum post concisely explaining what's going on and detailing how the proposals will be reviewed.  That post is called a "roadmap", and this field should link to it.  Most proposals don't need roadmaps.
+
+`Bug` should be used when this proposal is fixing a bug with significant discussion in the bug report.  It is not necessary to link bugs that do not contain significant discussion or that merely duplicate discussion linked somewhere else.  Do not link bugs from private bug trackers.
+
+`Implementation` should link to the PR(s) implementing the feature.  If the proposal has not been implemented yet, or if it simply codifies existing behavior, just say that.  If the implementation has already been committed to the main branch (as an experimental feature), say that and specify the experimental feature flag.  If the implementation is spread across multiple PRs, just link to the most important ones.
+
+`Feature Identifier` should be the feature name used to identify this feature under [SE-0362](https://github.com/apple/swift-evolution/blob/main/proposals/0362-piecemeal-future-features.md#proposals-define-their-own-feature-identifier).  Not all proposals need a feature identifier.  You should think about whether one would be useful for your proposal as part of filling this field out.
+
+`Previous Proposal` should be used when there is a specific line of succession between the linked proposal and this one.  For example, this proposal might have been removed from the previous proposal so that it can be reviewed separately, or this proposal might supersede the previous proposal in some way that was felt to exceed the scope of a "revision".
+
+`Previous Revision` should be added after a major substantive revision of a proposal that has undergone review.  It links to the previously reviewed revision.  It is not necessary to add or update this field after minor editorial changes.
+
+`Review` is a history of all discussion threads about this proposal, in chronological order.  Use these standardized link names: `pitch` `review` `revision` `acceptance` `rejection`.  If there are multiple such threads, spell the ordinal out: `first pitch` `second review` etc.
```

Comment:
The rest of the template is hard-wrapped, but this addition is soft wrapped.  We should probably match the existing content.

Reply:
Done, thanks!